modules/core/shared/src/main/scala/lucuma/core/enums/GmosNorthGrating.scala

```diff
@@ -34,7 +36,7 @@ sealed abstract class GmosNorthGrating(
 
   /** Compute the coverage of this disperser, given a central wavelength. */
   def coverage(λ: Wavelength): Coverage =
-    Coverage.centered(λ, simultaneousCoverage)
+    Coverage.centered(λ, simultaneousCoverage.tToUnit[Picometer])
```

Comment:
This is an example of using `tToUnit`. I'm not sure what does it mean i practice to truncate

Reply:
`simultaneousCoverage` is expressed in integer nanometers. We cannot change the unit to picometers without loss of precision i.e. truncation.